packages/common/src/ConfigResponse.ts

```diff
@@ -0,0 +1,59 @@
+import {
+  Address,
+  IntString,
+  NpmLogLevel
+} from './types/Aliases'
+import { Environment } from './Environments'
+
+export interface LoggerConfiguration {
+  logLevel: NpmLogLevel
+  loggerUrl?: string
+  userId?: string
+  applicationId?: string
+}
+
+export interface ConfigResponse {
+  networks: Record<number|string, ConfigEntry>
```

Comment:
What do you need a 'Record' for?

Reply:
Why not?

Reply:
Will it work with [key: number]: ConfigEntry